app/script/view_model/AuthViewModel.coffee

```diff
@@ -1251,6 +1252,7 @@ class z.ViewModel.AuthViewModel
       return @client_repository.get_valid_local_client()
     .catch (error) =>
       @logger.info "No valid local client found: #{error.message}", error
+      @storage_repository.delete_client_crypto()
```

Comment:
Prevent mixed crypto state: This is where the new crypto was generated while old sessions were kept.